chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -272,8 +272,23 @@
   realm = nil
 end
 
-template "/opt/dell/crowbar_framework/rainbows.cfg" do
-  source "rainbows.cfg.erb"
+directory "/etc/sysconfig/crowbar" do
```

Comment:
This is wrong, `/etc/sysconfig/crowbar` should be the file and not a directory :)